package.json

```diff
@@ -23,6 +23,7 @@
     "projects/landing-page"
   ],
   "scripts": {
+    "downloadSpecs": "node downloadSpecs.js",
```

Comment:
When I run this after checking out the pull request I get:
```
λ ~/code/paritytech/substrate-connect/ git nik-fix-specs-location  » yarn downloadSpecs
yarn run v1.22.10
$ node downloadSpecs.js
Init downloading of specs for chains polkadot, kusama, westend.
Downloading 'polkadot.json' in '.chains' dir
Downloading 'kusama.json' in '.chains' dir
Downloading 'westend.json' in '.chains' dir
Successfully moved '.chains/polkadot.json' to 'packages/connect/assets'
Successfully moved '.chains/kusama.json' to 'packages/connect/assets'
Successfully moved '.chains/westend.json' to 'packages/connect/assets'
Successfully moved '.chains/polkadot.json' to 'projects/extension/public/assets'
Successfully moved '.chains/kusama.json' to 'projects/extension/public/assets'
Successfully moved '.chains/westend.json' to 'projects/extension/public/assets'
/home/raoul/code/paritytech/substrate-connect/downloadSpecs.js:32
        if (err) throw err
                 ^

[Error: ENOENT: no such file or directory, copyfile '.chains/polkadot.json' -> 'projects/extension/public/assets/polkadot.json'] {
  errno: -2,
  code: 'ENOENT',
  syscall: 'copyfile',
  path: '.chains/polkadot.json',
  dest: 'projects/extension/public/assets/polkadot.json'
}
error Command failed with exit code 1.
```

Reply:
That was due to empty dirs (when you gitignore everything in a dir - git ignores the dir itself). It is fixed now in latest pushes.